modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -584,6 +584,11 @@ module.exports = (self) => {
         destination[field.name] = null;
         return;
       }
+      if (!newDateVal && !field.def){
+        // If no inputted date or default date, leave as empty
+        destination[field.name] = null
```

Comment:
👍 I'm OK with this. Technically it's a change of behavior, but I do not think this behavior served anyone on the server side, where it was very unexpected. Please add a space after `field.def)` and before the `{`

Reply:
Also make sure the unit tests still pass (`npm test`) if not we might need to adjust one to account for not automatically filling in today's date.